### ISSUE TITLE
Appointments does not show available slots

### ISSUE THREAD

**Issue by Tealk** [nextcloud/calendar#3896]
### Steps to reproduce
Open new appointment
choose one of my calendars
set the time
click save

### Expected behaviour
I can select dates in free times

### Actual behaviour
I do not get any dates displayed. Even if the whole day is free, nothing is displayed.
If I select another calendar it works but

## Server configuration detail

**Operating system:** Linux 5.11.22-5-pve #1 SMP PVE 5.11.22-10 (Tue, 28 Sep 2021 08:15:41 +0200) x86_64

**Webserver:** nginx/1.18.0 (fpm-fcgi)

**Database:** mysql 10.5.12

**PHP version:** 

8.0.14
Modules loaded: Core, date, libxml, openssl, pcre, zlib, filter, hash, json, Reflection, SPL, session, standard, sodium, cgi-fcgi, mysqlnd, PDO, xml, apcu, bcmath, calendar, ctype, curl, dom, mbstring, FFI, fileinfo, ftp, gd, gettext, gmp, iconv, igbinary, imagick, imap, intl, exif, mysqli, pdo_mysql, Phar, posix, readline, redis, shmop, SimpleXML, sockets, sysvmsg, sysvsem, sysvshm, tidy, tokenizer, xmlreader, xmlwriter, xsl, zip, Zend OPcache

**Nextcloud version:** 23.0.0 - 23.0.0.10

**Updated from an older Nextcloud/ownCloud or fresh install:** 

**Where did you install Nextcloud from:** unknown

<details><summary>Signing status</summary>

Array
(
    [core] => Array
        (
            [EXTRA_FILE] => Array
                (
                    [upgrade.log] => Array
                        (
                            [expected] => 
                            [current] => cf3f7f41ec2ea7fc1074df4fcc614985664961b07091868e99cdfe02e249e9c2f25f0afba2d98a337b4cf33c0d2c0d8bb4b6309ccf51dc77559ff8099f0bf764
                        )

                )

        )

)

</details>

<details><summary>List of activated apps</summary>

```
Enabled:
 - accessibility: 1.9.0
 - activity: 2.15.0
 - admin_audit: 1.13.0
 - apporder: 0.14.0
 - approval: 1.0.9
 - breezedark: 23.1.0
 - calendar: 3.0.5
 - circles: 23.0.0
 - cloud_federation_api: 1.6.0
 - comments: 1.13.0
 - contacts: 4.0.7
 - dashboard: 7.3.0
 - dav: 1.21.0
 - deck: 1.6.0
 - drop_account: 2.0.0
 - event_update_notification: 1.4.0
 - federatedfilesharing: 1.13.0
 - federation: 1.13.0
 - files: 1.18.0
 - files_downloadactivity: 1.12.0
 - files_linkeditor: 1.1.8
 - files_markdown: 2.3.5
 - files_pdfviewer: 2.4.0
 - files_rightclick: 1.2.0
 - files_sharing: 1.15.0
 - files_texteditor: 2.14.0
 - files_trashbin: 1.13.0
 - files_versions: 1.16.0
 - files_videoplayer: 1.12.0
 - fileslibreofficeedit: 1.0.4
 - firstrunwizard: 2.12.0
 - forms: 2.4.0
 - groupfolders: 11.1.2
 - guests: 2.1.0
 - imageconverter: 1.3.2
 - issuetemplate: 0.7.0
 - jitsi: 0.12.0
 - logreader: 2.8.0
 - lookup_server_connector: 1.11.0
 - mail: 1.11.5
 - maps: 0.1.10
 - metadata: 0.15.0
 - news: 17.0.1
 - nextcloud_announcements: 1.12.0
 - notes: 4.3.0
 - notifications: 2.11.1
 - notify_push: 0.3.0
 - oauth2: 1.11.0
 - onlyoffice: 7.3.0
 - password_policy: 1.13.0
 - passwords: 2022.1.20
 - photos: 1.5.0
 - polls: 3.4.2
 - privacy: 1.7.0
 - provisioning_api: 1.13.0
 - quota_warning: 1.13.0
 - registration: 1.4.0
 - serverinfo: 1.13.0
 - settings: 1.5.0
 - sharebymail: 1.13.0
 - socialsharing_email: 2.4.0
 - spreed: 13.0.1
 - survey_client: 1.11.0
 - systemtags: 1.13.0
 - tasks: 0.14.2
 - text: 3.4.0
 - theming: 1.14.0
 - theming_customcss: 1.10.0
 - twofactor_admin: 3.2.0
 - twofactor_backupcodes: 1.12.0
 - twofactor_email: 2.1.1
 - twofactor_nextcloud_notification: 3.3.1
 - twofactor_totp: 6.2.0
 - twofactor_u2f: 6.3.0
 - updatenotification: 1.13.0
 - uppush: 0.3.0
 - user_retention: 1.6.0
 - user_status: 1.3.1
 - viewer: 1.7.0
 - workflowengine: 2.5.0
Disabled:
 - appointments
 - contactsinteraction
 - encryption
 - files_external
 - integration_mastodon
 - integration_whiteboard
 - recommendations
 - support
 - twofactor_gateway
 - user_ldap
 - weather_status

```
</details>

<details><summary>Configuration (config/config.php)</summary>

```
{
    "instanceid": "***REMOVED SENSITIVE VALUE***",
    "passwordsalt": "***REMOVED SENSITIVE VALUE***",
    "secret": "***REMOVED SENSITIVE VALUE***",
    "trusted_domains": [
        "anzah.cloud",
        "rollenspiel.cloud",
        "localhost"
    ],
    "datadirectory": "***REMOVED SENSITIVE VALUE***",
    "overwrite.cli.url": "http:\/\/anzah.cloud",
    "htaccess.RewriteBase": "\/",
    "dbtype": "mysql",
    "version": "23.0.0.10",
    "dbname": "***REMOVED SENSITIVE VALUE***",
    "dbhost": "***REMOVED SENSITIVE VALUE***",
    "dbport": "",
    "dbtableprefix": "oc_",
    "dbuser": "***REMOVED SENSITIVE VALUE***",
    "dbpassword": "***REMOVED SENSITIVE VALUE***",
    "mysql.utf8mb4": true,
    "installed": true,
    "maintenance": false,
    "default_language": "de",
    "default_locale": "de",
    "default_phone_region": "DE",
    "allow_user_to_change_display_name": true,
    "knowledgebaseenabled": true,
    "theme": "",
    "defaultapp": "apporder",
    "skeletondirectory": "\/mnt\/HDD\/skeleton",
    "trashbin_retention_obligation": "120, auto",
    "versions_retention_obligation": "120, auto",
    "appcodechecker": true,
    "updatechecker": true,
    "updater.release.channel": "stable",
    "check_for_working_webdav": true,
    "loglevel": 0,
    "logdateformat": "d.m.Y H:i:s",
    "logtimezone": "Europe\/Berlin",
    "log_rotate_size": 10485760,
    "mail_smtpmode": "smtp",
    "mail_smtpauthtype": "PLAIN",
    "mail_from_address": "***REMOVED SENSITIVE VALUE***",
    "mail_domain": "***REMOVED SENSITIVE VALUE***",
    "mail_smtpsecure": "ssl",
    "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
    "mail_smtpport": "465",
    "mail_smtpauth": 1,
    "mail_smtpname": "***REMOVED SENSITIVE VALUE***",
    "mail_smtppassword": "***REMOVED SENSITIVE VALUE***",
    "enable_previews": true,
    "preview_max_x": 2048,
    "preview_max_y": 2048,
    "preview_max_filesize_image": 50,
    "preview_max_scale_factor": 10,
    "data-fingerprint": "01b9630d063ba580c331de332a0ab360",
    "memcache.local": "\\OC\\Memcache\\Redis",
    "memcache.distributed": "\\OC\\Memcache\\Redis",
    "memcache.locking": "\\OC\\Memcache\\Redis",
    "filelocking.enabled": "true",
    "redis": {
        "host": "***REMOVED SENSITIVE VALUE***",
        "port": 6379,
        "dbindex": 10,
        "password": "***REMOVED SENSITIVE VALUE***",
        "timeout": 1.5
    },
    "twofactor_enforced": false,
    "twofactor_enforced_groups": [
        "admin"
    ],
    "twofactor_enforced_excluded_groups": [],
    "mail_sendmailmode": "smtp",
    "app_install_overwrite": [
        "apporder",
        "circles",
        "groupfolders",
        "whiteboard",
        "issuetemplate",
        "tasks",
        "bookmarks",
        "appointments",
        "quota_warning",
        "jitsi",
        "passwords",
        "files_downloadactivity"
    ],
    "migrate_guest_user_data": true,
    "remove_guest_account_on_conversion": true,
    "keep_users_without_login": "no",
    "trusted_proxies": "***REMOVED SENSITIVE VALUE***"
}
```
</details>

**Are you using external storage, if yes which one:** local

**Are you using encryption:** 

**Are you using an external user-backend, if yes which one:** 

## Client configuration

**Browser:** Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:96.0) Gecko/20100101 Firefox/96.0

**Operating system:** Debian 11

## Logs

<details><summary>Web server error log</summary>

```
no Web server error
```
</details>

<details><summary>Nextcloud log</summary>

```
nothing in the Nextcloud logs
```
</details>

<details><summary>Browser log</summary>

nothing in the Browser log
</details>

**Comment by Tealk** [comment]
Yes sounds exactly like my problem

Me and the github search are not friends anymore it always hides results from me

**Comment by elhananjair** [comment]
> Possibly related to #3781 (fix in 23.0.1).

yeah, it is related to that issue but that is an issue that has been fixed at that time, but this one is from Jan 20, which means this issue still exists after that fix, and I am facing that too.
